deps/ngtcp2/lib/includes/config.h

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
note: This is necessary for building ngtcp2 on windows

Reply:
The space? lol

Reply:
No, silly, the file